libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -339,7 +339,7 @@ TEST(FixedPointTest, FromString_64_64)
   EXPECT_EQ(m_one_point_five, m_one_point_five_flt);
 
   fp128_t e1("2.718281828459045235360287471352662498");
-  fp128_t e2(2.718281828459045235360287471352662498);
+  fp128_t e2("2.718281828459045235360287471352662498");
   EXPECT_TRUE(e1.Near(e2));
   EXPECT_NE(e1, e2);
```

Comment:
I would expect e1 == e2 now? I'm not sure this test tests anything meaningful any more!

Reply:
I guess you're right, it's a mistake; I wouldn't touch this test then, let @markos review it later; it is not critical change anyway.